src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1890,14 +1894,14 @@ protected override void HandleEvaluateRequestAsync(IRequestResponder<EvaluateArg
                 flags |= enum_EVALFLAGS.EVAL_NOSIDEEFFECTS;
             }
 
-            if (context == EvaluateArguments.ContextValue.Clipboard)
-            {
-                dapEvalFlags |= DAPEvalFlags.CLIPBOARD_CONTEXT;
-            }
-
             IDebugProperty2 property;
             if (expressionObject is IDebugExpressionDAP expressionDapObject)
             {
+                if (context == EvaluateArguments.ContextValue.Clipboard)
+                {
+                    dapEvalFlags |= DAPEvalFlags.CLIPBOARD_CONTEXT;
```

Comment:
Scope dapEvalFlags to the context of the `if` statement here. its not necessary to have it scoped outside of this as it isn't used except for the call below.